### DISCUSSION TITLE
Shop category template

### DISCUSSION THREAD

**Discussion by haperstone** [bootscore/bootscore#457] [Q&A]
I don’t see a product category template in the theme, how can I customize the category?

**Comment by crftwrk** [comment]
Category uses `woocommerce.php`. If you explain what exactly do you want to do, we'll help you out.

**Comment by haperstone** [comment]
Thank U for fast response. yes, I understand that template is used for all woocommerce pages. For example, I need to customize only product category pages (archives).

**Reply by crftwrk** [reply]
You cannot edit the category, but you can edit the loop. That's why I'm asking what exactly do you want to do. If I know your plan, I can guide you through.

**Reply by haperstone** [reply]
I want to add a custom filter to product categories

**Reply by crftwrk** [reply]
You can add the Woo product filter widget to the sidebar.

**Reply by haperstone** [reply]
i dont want use the sidebar, i wnat add filter to title area. Also i want add description with image after products. It's why i want create my own category template.

**Reply by crftwrk** [reply]
As mentioned in previous comment, theme uses `woocommerce.php` and that's why you cannot edit `archive-product.php`. See here https://woocommerce.com/document/woocommerce-theme-developer-handbook/#section-2
 
Here are 4 ideas how to create something you want:

1. Use category description. You can insert plain HTML into the description and shortcodes work there as well. Don't know how your custom filter works, but if it's just HTML or a shortcode widget, it will work.
2. Add a new widget position wherever you want in `woocommerce.php`, add your stuff here. Install https://widgetcontext.com and assign widgets only to this category.
3. Create a usual page, add your content and use shortcode `[products category="shoes"]` to show the loop.
4. Try to remove `woocommerce.php` in main-theme and use hooks https://woocommerce.com/document/woocommerce-theme-developer-handbook/#section-3. This may a bit tricky, but if that works well we can merge this here and remove `woocommerce.php`. Now you can edit `archive-product.php`.

Does this help?

**Reply by haperstone** [reply]
Thank You. Solution with deleting woocommerce.php works for me
